packages/typespec-python/test/generated/azure-core-lro-rpc/_specs_/azure/core/lro/rpc/aio/_operations/_operations.py

```diff
@@ -458,16 +452,16 @@ async def begin_create_job_final_on_location(
         :paramtype polling: bool or ~azure.core.polling.AsyncPollingMethod
         :keyword int polling_interval: Default waiting time between two polls for LRO operations if no
          Retry-After header is present.
-        :return: An instance of AsyncLROPoller that returns JobResult. The JobResult is compatible with
-         MutableMapping
-        :rtype: ~azure.core.polling.AsyncLROPoller[~_specs_.azure.core.lro.rpc.models.JobResult]
+        :return: An instance of AsyncLROPoller that returns JSON object. The JSON object is compatible
+         with MutableMapping
+        :rtype: ~azure.core.polling.AsyncLROPoller[JSON]
         :raises ~azure.core.exceptions.HttpResponseError:
         """
 
     @distributed_trace_async
     async def begin_create_job_final_on_location(
         self, body: Union[_models.JobData, JSON, IO], **kwargs: Any
-    ) -> AsyncLROPoller[_models.JobResult]:
+    ) -> AsyncLROPoller[JSON]:
```

Comment:
why are we switching to JSON everywhere?

Reply:
`getLroMetadata` can't give right response type for rpc test because of https://github.com/Azure/typespec-azure/issues/2862. And in scrum meeting we decide to wait for its fix instead of [workaround](https://github.com/Azure/autorest.python/pull/1821/commits/cf7dc848bd0a46da89c9e67af5c331b7dcf6115e#diff-c06d9ac822f471f9231e983bb5f2b9ae51fe8fbc2fc6f45b4cf9418d87e839ba).